### DISCUSSION TITLE
Authentication bolierplate module

### DISCUSSION THREAD

**Discussion by feelixe** [nuxt/nuxt#17930] [General]
Hello, just some of my thoughts regarding authentication.

I think `@nuxt/auth` was one of the best things about Nuxt v2.

With ohmyfetch implemented in Nuxt v3 and the new `useState` I think it would be a missed opportunity to not have a module to combine these into a similar package of `@nuxt/auth`. I guess the old auth package could work with Nuxt v3 with some work, but that would force the usage of both Axios and VueX which is a huge drawback for someone like me who prefers to stay within the framework defaults as much as possible.

I think a module like this can be a make or break for a lot of developers who are testing out Nuxt for the first time.

I would love to contribute to such a module but I'm not really experienced enough to start it.

Best regards

**Comment by dodanex** [comment]
Agree! An official nuxt authentication plugin is a must have because we use authentication in almost every project.
